scripts/llnl_scripts/llnl_lc_build_tools.py

```diff
@@ -659,8 +659,6 @@ def get_specs_for_current_machine():
     else:
         specs = specs_json[sys_type]
 
-    specs = ['%' + spec for spec in specs]
```

Comment:
Removed because specs are now defined as "+variants %compiler ^dependency_restrictions". The added "%" at the beginning causes error.